### ISSUE TITLE
[crash] 1654+_oregon-core (BB #491)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#491]
This issue was migrated from bitbucket.
**Original Reporter:** 
**Original Date:** 03.02.2014 02:22:56 GMT+0000
**Original Priority:** critical
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491

<hr>

Game crash on last rev.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 03.02.2014 05:10:07 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8051175

<hr>

Latest revision? Pull request not included?

It just looks to me like it's something related with some of Zaffys recent work. And that's not merged into the official repository yet. So are you using OregonCore pulled from this repo?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 03.02.2014 05:41:17 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8051376

<hr>

Scratch that.

This happens when you open the who tab or press 'O' right?

Can I confirm some things... what's your native language? What's the language on the computer? What kind of client are you using? (enGB? enUS?)

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 03.02.2014 05:49:05 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8051430

<hr>

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 03.02.2014 17:04:09 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8062597

<hr>

Looks like duplicate of #403, although those question asked by LordUsagi are VERY important to us.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** digger
**Original Date:** 03.02.2014 23:00:15 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8067914

<hr>

I'm sorry, but my English is too bad to explain. 
Kernel was compiled in debug here "https://bitbucket.org/oregon/oregoncore/src" + two patches "pvprank.patch" and "renamenpc.patch" 
crash dump was posted by me above.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 03.02.2014 23:54:08 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8068421

<hr>

All right. How you reproduce crash?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 03.02.2014 23:58:39 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8068464

<hr>

Can you give us some examples of names your characters have ?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** digger
**Original Date:** 03.02.2014 23:59:52 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8068469

<hr>

I do not know how to intentionally and with a 100% chance to play it, but the server is running with this kernel works for 5-10 minutes.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 04.02.2014 00:04:20 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8068500

<hr>

Press O and look at your "Who" list and tell me if crash happens

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** digger
**Original Date:** 04.02.2014 00:05:01 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8068505

<hr>

some names "trap" and "zizi"

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 04.02.2014 00:05:33 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8068510

<hr>

Please set up your `WorldLogFile` config in `oregoncore.conf` and then wait for the crash and paste the output here.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 04.02.2014 00:08:31 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8068530

<hr>

You can also [click here](https://chat.mibbit.com/?url=irc%3A%2F%2Firc.foonetic.net%2FOregonCore) to get instant access to our IRC channel so we can communicate more effectively.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** digger
**Original Date:** 04.02.2014 00:24:04 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8068644

<hr>

You're right, if you press the "O" and then the "who" and after you press the "update" crash occurs.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** digger
**Original Date:** 04.02.2014 00:27:09 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8068670

<hr>

Server.log

```

2014-02-04 12:19:27 Allowed Level: 0 Result 0
2014-02-04 12:19:27 Allowed Level: 0 Player Level 0
2014-02-04 12:19:27 Server side warden for client 2 initializing...
2014-02-04 12:19:27   C->S Key: 91 B2 21 30 C4 39 EC 0A A2 AE 70 59 DE 1F 8A A8 
2014-02-04 12:19:27   S->C Key: 7C 94 FC 3C F7 FE A3 DE 2B 9B 11 58 37 E2 3C CE 
2014-02-04 12:19:27   Seed: 4D 80 8D 2C 77 D9 05 C4 1A 63 80 EC 08 58 6A FE 
2014-02-04 12:19:27 Loading Module...
2014-02-04 12:19:27   Module Key: AE 25 BC 51 06 3B 77 BD 36 3C 3E FE 0F C1 73 F9 
2014-02-04 12:19:27   Module ID: 79 C0 76 8D 65 79 77 D6 97 E1 0B AD 95 6C CE D1 
2014-02-04 12:19:27 Request module
2014-02-04 12:19:27 Server Population (0.020000).
2014-02-04 12:19:27 Got packet, opcode 01, size 1
2014-02-04 12:19:27 STORAGE_SIZE: 1
01 
2014-02-04 12:19:27 Request hash
2014-02-04 12:19:27 Got packet, opcode 04, size 21
2014-02-04 12:19:27 STORAGE_SIZE: 21
04 56 8C 05 4C 78 1A 97 | 2A 60 37 A2 29 0C 22 B5 
25 71 A0 6F 4E 
2014-02-04 12:19:27 Request hash reply: succeed
2014-02-04 12:19:27 Initialize module
2014-02-04 12:19:28 Loading char guid 373 from account 2.
2014-02-04 12:19:28 Loading char guid 440 from account 2.
2014-02-04 12:19:28 Loading char guid 2143 from account 2.
2014-02-04 12:19:32 WORLD: CMSG_VOICE_SETTINGS
2014-02-04 12:19:32 STORAGE_SIZE: 2
00 01 
2014-02-04 12:19:32 WORLD: CMSG_VOICE_SETTINGS
2014-02-04 12:19:32 STORAGE_SIZE: 2
00 01 
2014-02-04 12:19:32 WORLD: CMSG_VOICE_SETTINGS
2014-02-04 12:19:32 STORAGE_SIZE: 2
00 01 
2014-02-04 12:19:32 WORLD: Received CMSG_GROUP_PASS_ON_LOOT
2014-02-04 12:19:37 Request data
2014-02-04 12:19:37 STORAGE_SIZE: 25
02 00 28 8C 00 DE 98 7B | 00 02 8C 00 5B 84 8C 00 
05 8C 00 0B 06 89 00 05 | 7F 
2014-02-04 12:19:37 Sent check id's: 4 3 2 
2014-02-04 12:19:42 Allowed Level: 0 Result 0
2014-02-04 12:19:42 Allowed Level: 0 Player Level 0
2014-02-04 12:19:42 Server side warden for client 4513 initializing...
2014-02-04 12:19:42   C->S Key: 16 45 D1 B5 EC 59 D2 F3 1B AC 56 0A 26 50 B5 8E 
2014-02-04 12:19:42   S->C Key: 99 23 3E 1A 91 F3 7E 2C 39 D0 8D 4D 32 05 FE 76 
2014-02-04 12:19:42   Seed: 4D 80 8D 2C 77 D9 05 C4 1A 63 80 EC 08 58 6A FE 
2014-02-04 12:19:42 Loading Module...
2014-02-04 12:19:42   Module Key: AE 25 BC 51 06 3B 77 BD 36 3C 3E FE 0F C1 73 F9 
2014-02-04 12:19:42   Module ID: 79 C0 76 8D 65 79 77 D6 97 E1 0B AD 95 6C CE D1 
2014-02-04 12:19:42 Request module
2014-02-04 12:19:42 Server Population (0.040000).
2014-02-04 12:19:44 Got packet, opcode 01, size 1
2014-02-04 12:19:44 STORAGE_SIZE: 1
01 
2014-02-04 12:19:44 Request hash
2014-02-04 12:19:44 Loading char guid 2076 from account 4513.
2014-02-04 12:19:44 Loading char guid 2224 from account 4513.
2014-02-04 12:19:45 Got packet, opcode 04, size 21
2014-02-04 12:19:45 STORAGE_SIZE: 21
04 56 8C 05 4C 78 1A 97 | 2A 60 37 A2 29 0C 22 B5 
25 71 A0 6F 4E 
2014-02-04 12:19:45 Request hash reply: succeed
2014-02-04 12:19:45 Initialize module
2014-02-04 12:19:50 Added aura spellid 34074, effect 0
2014-02-04 12:19:50 Added aura spellid 24529, effect 0
2014-02-04 12:19:50 Added aura spellid 35061, effect 0
2014-02-04 12:19:50 Creating grid[39,36] for map 530 instance 0
2014-02-04 12:19:50 Loading map ./maps/5302427.map
2014-02-04 12:19:50 VMAP loaded name:Запределье, id:530, x:24, y:27 (vmap rep.: x:24, y:27)

2014-02-04 12:19:50 CHARACTER: Sent Initial Spells
2014-02-04 12:19:50 Initializing Action Buttons for '2143'
2014-02-04 12:19:50 Action Buttons for '2143' Initialized

2014-02-04 12:19:50 39 GameObjects, 174 Creatures, and 0 Corpses/Bones loaded for grid 2532 on map 530
2014-02-04 12:19:50 Creating player data for himself 2143
2014-02-04 12:19:50 WORLD: Sent SMSG_CONTACT_LIST
2014-02-04 12:19:50 WORLD: Spell Effect DUMMY
2014-02-04 12:19:50 Creating grid[31,31] for map 530 instance 0
2014-02-04 12:19:50 Loading map ./maps/5303232.map
2014-02-04 12:19:50 VMAP loaded name:Запределье, id:530, x:32, y:32 (vmap rep.: x:32, y:32)
2014-02-04 12:19:50 SetMinion 22123 for 0, apply 1
2014-02-04 12:19:50 New Pet has guid 0
2014-02-04 12:19:50 Got packet, opcode 02, size 27
2014-02-04 12:19:50 STORAGE_SIZE: 27
02 14 00 05 43 1E 39 01 | A0 69 5E 13 00 75 41 00 
C0 85 4A 33 40 00 E0 4D | 62 50 3F 
2014-02-04 12:19:50 Handle data
2014-02-04 12:19:50 CHECKSUM IS VALID
2014-02-04 12:19:50 ServerTicks 689676359
2014-02-04 12:19:50 RequestTicks 689663296
2014-02-04 12:19:50 Ticks 324954528
2014-02-04 12:19:50 Ticks diff 13063
2014-02-04 12:19:50 RESULT MEM_CHECK passed CheckId 1003 account Id 2
2014-02-04 12:19:50 RESULT MEM_CHECK passed CheckId 1002 account Id 2
2014-02-04 12:19:50 RESULT MEM_CHECK passed CheckId 1001 account Id 2
2014-02-04 12:19:50 WORLD: Sent (SMSG_GUILD_QUERY_RESPONSE)
2014-02-04 12:19:50 Grid[31,31] on map 530 moved to REMOVAL state
2014-02-04 12:19:51 WORLD: Battleground status
2014-02-04 12:19:51 WORLD: Received (MSG_GUILD_BANK_MONEY_WITHDRAWN)
2014-02-04 12:19:51 WORLD: Sent MSG_GUILD_BANK_MONEY_WITHDRAWN
2014-02-04 12:19:51 WORLD: Received (MSG_GUILD_PERMISSIONS)
2014-02-04 12:19:51 WORLD: Sent (MSG_GUILD_PERMISSIONS)
2014-02-04 12:19:51 WORLD: CMSG_VOICE_SETTINGS
2014-02-04 12:19:51 STORAGE_SIZE: 2
00 01 
2014-02-04 12:19:51 WORLD: CMSG_VOICE_SETTINGS
2014-02-04 12:19:51 STORAGE_SIZE: 2
00 01 
2014-02-04 12:19:51 HandlePetNameQuery. CMSG_PET_NAME_QUERY
2014-02-04 12:19:51 Opcode 151
2014-02-04 12:19:51 Opcode 151
2014-02-04 12:19:51 Opcode 151
2014-02-04 12:19:51 Opcode 151
2014-02-04 12:19:51 WORLD: CMSG_VOICE_SETTINGS
2014-02-04 12:19:51 STORAGE_SIZE: 2
00 01 
2014-02-04 12:19:51 Opcode 979
2014-02-04 12:19:51 Opcode 979
2014-02-04 12:19:51 Opcode 979
2014-02-04 12:19:51 Opcode 979
2014-02-04 12:19:51 WORLD: Received CMSG_GROUP_PASS_ON_LOOT
2014-02-04 12:19:51 deal dmg:61 to health:6542 
2014-02-04 12:19:51 deal dmg:0 to health:6383 
2014-02-04 12:19:51 deal dmg:159 to health:6481 
2014-02-04 12:19:51 creature 72451 stopped attacking player 2143
2014-02-04 12:19:52 deal dmg:131 to health:6322 
2014-02-04 12:19:52 deal dmg:156 to health:6191 
2014-02-04 12:19:53 deal dmg:73 to health:6035 
2014-02-04 12:19:53 deal dmg:119 to health:4567 
2014-02-04 12:19:54 deal dmg:277 to health:5962 
2014-02-04 12:19:54 deal dmg:184 to health:5685 
2014-02-04 12:19:55 deal dmg:168 to health:5501 
2014-02-04 12:19:55 Request data
2014-02-04 12:19:55 STORAGE_SIZE: 25
02 00 28 8C 00 DE 98 7B | 00 02 8C 00 5B 84 8C 00 
05 8C 00 0B 06 89 00 05 | 7F 
2014-02-04 12:19:55 Sent check id's: 4 3 2 
2014-02-04 12:19:55 deal dmg:118 to health:4448 
2014-02-04 12:19:57 deal dmg:167 to health:5333 
2014-02-04 12:19:57 deal dmg:251 to health:5166 
2014-02-04 12:19:57 WORLD: Sent SMSG_CONTACT_LIST
2014-02-04 12:19:57 WORLD: Sent SMSG_CONTACT_LIST
2014-02-04 12:19:57 deal dmg:75 to health:4915 
2014-02-04 12:19:57 deal dmg:92 to health:4330 
2014-02-04 12:19:58 deal dmg:208 to health:4840 
2014-02-04 12:19:58 deal dmg:157 to health:4632 
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** digger
**Original Date:** 04.02.2014 00:28:44 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8068677

<hr>

New crash log:

```
Revision: OregonCore Rev: 1654+ Hash: ac0577b6a549+ (Win64,little-endian)
Date 4:2:2014. Time 12:20 
//=====================================================
*** Hardware ***
Processor: Intel(R) Xeon(R) CPU           E5405  @ 2.00GHz
Number Of Processors: 8
Physical Memory: 8387256 KB (Available: 5801108 KB)
Commit Charge Limit: 12126724 KB

*** Operation System ***
Microsoft Windows Server 2003 Enterprise Edition Service Pack 2 (Version 5.2, Build 3790)

//=====================================================
Exception code: 80000003 BREAKPOINT
Fault address:  0000000000C3CEB5 01:000000000006BEB5 W:\WoW_BC_2.4.3\libmysql.dll

Registers:
RAX:0000000000000000
RBX:0000000010CD52A0
RCX:0000000000000016
RDX:0000000000000000
RSI:0000000000000000
RDI:000000002EA6E070
R8: 0000000000000000
R9: 0000000000000000
R10:0000000000000202
R11:0000000000000000
R12:0000000000000000
R13:0000000000000000
R14:0000000000000000
R15:CCCCCCCCCCCCCCCC
CS:RIP:0033:0000000000C3CEB5
SS:RSP:002B:000000002EA64C30  RBP:00000000
DS:002B  ES:002B  FS:0053  GS:002B
Flags:00000212

Call stack:
Address   Frame     Function      SourceFile
0000000000C3CEB5  000000002EA64C30  my_sigabrt_handler+5  d:\compilling_wow_2.4.3\dep\mysqllite\mysys\my_thr_init.c line 520
000000001031959D  000000002EA64CB0  raise+32D
000000001031267B  000000002EA66FF0  _CrtDbgReportW+67B
000000001031BEAC  000000002EA6F100  _VCrtDbgReportW+B4C
00000000103120B3  000000002EA6F140  _CrtDbgReportW+B3
000000001031204D  000000002EA6F190  _CrtDbgReportW+4D
00000000104D7205  000000002EA6F1D0  ?_Debug_message@std@@YAXPEB_W0I@Z+35
0000000140076FCD  000000002EA6F200  std::basic_string<wchar_t,std::char_traits<wchar_t>,std::allocator<wchar_t> >::operator[]+4D  c:\program files (x86)\microsoft visual studio 10.0\vc\include\xstring line 1447
00000001407899F5  000000002EA6F290  Utf8toWStr+B5  d:\compilling_wow_2.4.3\src\shared\util.cpp line 312
0000000140736CD9  000000002EA6F990  WorldSession::HandleWhoOpcode+2F9  d:\compilling_wow_2.4.3\src\game\mischandler.cpp line 202
000000014024F916  000000002EA6F9D0  WorldSession::ExecuteOpcode+56  d:\compilling_wow_2.4.3\src\game\worldsession.cpp line 587
000000014024E1E7  000000002EA6FAB0  WorldSession::Update+207  d:\compilling_wow_2.4.3\src\game\worldsession.cpp line 218
00000001402B26D8  000000002EA6FBF0  World::UpdateSessions+158  d:\compilling_wow_2.4.3\src\game\world.cpp line 2255
00000001402AFB24  000000002EA6FD80  World::Update+394  d:\compilling_wow_2.4.3\src\game\world.cpp line 1699
0000000140019596  000000002EA6FDD0  WorldRunnable::run+A6  d:\compilling_wow_2.4.3\src\oregoncore\worldrunnable.cpp line 54
00000001407A2908  000000002EA6FE10  ACE_Based::Thread::ThreadTask+38  d:\compilling_wow_2.4.3\src\shared\threading.cpp line 187
0000000180114E89  000000002EA6FEA0  ACE_OS_Thread_Adapter::invoke+D9  d:\compilling_wow_2.4.3\dep\acelite\ace\os_thread_adapter.cpp line 86
0000000180054BD8  000000002EA6FEE0  ace_thread_adapter+38  d:\compilling_wow_2.4.3\dep\acelite\ace\base_thread_adapter.cpp line 120
00000000102072E5  000000002EA6FF30  _beginthreadex+2D5
00000000102072A4  000000002EA6FF70  _beginthreadex+294
0000000078D6B6DA  000000002EA6FFA0  BaseThreadStart+3A

Call stack:
Address   Frame     Function      SourceFile
0000000078EF0A2A  00000000009BF500  ZwWaitForSingleObject+A
0000000078D704BF  00000000009BF5A0  WaitForSingleObjectEx+DF
00000001801065BD  00000000009BF5F0  ACE_OS::thr_join+4D  d:\compilling_wow_2.4.3\dep\acelite\ace\os_ns_thread.inl line 2798
0000000180183182  00000000009BF620  ACE_Thread::join+32  d:\compilling_wow_2.4.3\dep\acelite\ace\thread.inl line 138
000000018018AA6D  00000000009BF7F0  ACE_Thread_Manager::wait_task+44D  d:\compilling_wow_2.4.3\dep\acelite\ace\thread_manager.cpp line 1793
00000001801822E4  00000000009BF820  ACE_Task_Base::wait+44  d:\compilling_wow_2.4.3\dep\acelite\ace\task.cpp line 86
00000001402E6469  00000000009BF850  ReactorRunnable::Wait+29  d:\compilling_wow_2.4.3\src\game\worldsocketmgr.cpp line 97
00000001402E5DCF  00000000009BF890  WorldSocketMgr::Wait+6F  d:\compilling_wow_2.4.3\src\game\worldsocketmgr.cpp line 305
0000000140010945  00000000009BFCE0  Master::Run+925  d:\compilling_wow_2.4.3\src\oregoncore\master.cpp line 239
000000014000FCEB  00000000009BFE20  ace_main_i+4BB  d:\compilling_wow_2.4.3\src\oregoncore\main.cpp line 166
000000014000FF35  00000000009BFE50  ACE_Main::run_i+35  d:\compilling_wow_2.4.3\src\oregoncore\main.cpp line 77
00000001800F5750  00000000009BFE80  ACE_Main_Base::run+40  d:\compilling_wow_2.4.3\dep\acelite\ace\os_main.cpp line 89
000000014000F7F9  00000000009BFED0  main+49  d:\compilling_wow_2.4.3\src\oregoncore\main.cpp line 77
0000000140A9BEEC  00000000009BFF40  __tmainCRTStartup+19C  f:\dd\vctools\crt_bld\self_64_amd64\crt\src\crtexe.c line 555
0000000140A9BD3E  00000000009BFF70  mainCRTStartup+E  f:\dd\vctools\crt_bld\self_64_amd64\crt\src\crtexe.c line 371
0000000078D5964C  00000000009BFFA0  BaseProcessStart+2C

Call stack:
Address   Frame     Function      SourceFile
0000000078EF0A2A  000000000374FA50  ZwWaitForSingleObject+A
0000000078D704BF  000000000374FAF0  WaitForSingleObjectEx+DF
0000000180109D2D  000000000374FB30  ACE_OS::cond_wait+AD  d:\compilling_wow_2.4.3\dep\acelite\ace\os_ns_thread.cpp line 1764
0000000180109A00  000000000374FBD0  ACE_OS::cond_timedwait+60  d:\compilling_wow_2.4.3\dep\acelite\ace\os_ns_thread.cpp line 1620
000000018006D98C  000000000374FC00  ACE_Condition_Thread_Mutex::wait+3C  d:\compilling_wow_2.4.3\dep\acelite\ace\condition_thread_mutex.cpp line 101
000000018006D9DB  000000000374FC30  ACE_Condition_Thread_Mutex::wait+3B  d:\compilling_wow_2.4.3\dep\acelite\ace\condition_thread_mutex.cpp line 108
000000018002867D  000000000374FC70  ACE_Message_Queue<ACE_MT_SYNCH>::wait_not_empty_cond+5D  d:\compilling_wow_2.4.3\dep\acelite\ace\message_queue_t.cpp line 1744
0000000180026100  000000000374FCE0  ACE_Message_Queue<ACE_MT_SYNCH>::dequeue_head+C0  d:\compilling_wow_2.4.3\dep\acelite\ace\message_queue_t.cpp line 1936
000000018003587E  000000000374FD30  ACE_Activation_Queue::dequeue+4E  d:\compilling_wow_2.4.3\dep\acelite\ace\activation_queue.cpp line 89
00000001407A9A55  000000000374FDD0  SqlDelayThread::run+85  d:\compilling_wow_2.4.3\src\shared\database\sqldelaythread.cpp line 39
00000001407A2908  000000000374FE10  ACE_Based::Thread::ThreadTask+38  d:\compilling_wow_2.4.3\src\shared\threading.cpp line 187
0000000180114E89  000000000374FEA0  ACE_OS_Thread_Adapter::invoke+D9  d:\compilling_wow_2.4.3\dep\acelite\ace\os_thread_adapter.cpp line 86
0000000180054BD8  000000000374FEE0  ace_thread_adapter+38  d:\compilling_wow_2.4.3\dep\acelite\ace\base_thread_adapter.cpp line 120
00000000102072E5  000000000374FF30  _beginthreadex+2D5
00000000102072A4  000000000374FF70  _beginthreadex+294
0000000078D6B6DA  000000000374FFA0  BaseThreadStart+3A

Call stack:
Address   Frame     Function      SourceFile
0000000078EF0A2A  000000000415FA50  ZwWaitForSingleObject+A
0000000078D704BF  000000000415FAF0  WaitForSingleObjectEx+DF
0000000180109D2D  000000000415FB30  ACE_OS::cond_wait+AD  d:\compilling_wow_2.4.3\dep\acelite\ace\os_ns_thread.cpp line 1764
0000000180109A00  000000000415FBD0  ACE_OS::cond_timedwait+60  d:\compilling_wow_2.4.3\dep\acelite\ace\os_ns_thread.cpp line 1620
000000018006D98C  000000000415FC00  ACE_Condition_Thread_Mutex::wait+3C  d:\compilling_wow_2.4.3\dep\acelite\ace\condition_thread_mutex.cpp line 101
000000018006D9DB  000000000415FC30  ACE_Condition_Thread_Mutex::wait+3B  d:\compilling_wow_2.4.3\dep\acelite\ace\condition_thread_mutex.cpp line 108
000000018002867D  000000000415FC70  ACE_Message_Queue<ACE_MT_SYNCH>::wait_not_empty_cond+5D  d:\compilling_wow_2.4.3\dep\acelite\ace\message_queue_t.cpp line 1744
0000000180026100  000000000415FCE0  ACE_Message_Queue<ACE_MT_SYNCH>::dequeue_head+C0  d:\compilling_wow_2.4.3\dep\acelite\ace\message_queue_t.cpp line 1936
000000018003587E  000000000415FD30  ACE_Activation_Queue::dequeue+4E  d:\compilling_wow_2.4.3\dep\acelite\ace\activation_queue.cpp line 89
00000001407A9A55  000000000415FDD0  SqlDelayThread::run+85  d:\compilling_wow_2.4.3\src\shared\database\sqldelaythread.cpp line 39
00000001407A2908  000000000415FE10  ACE_Based::Thread::ThreadTask+38  d:\compilling_wow_2.4.3\src\shared\threading.cpp line 187
0000000180114E89  000000000415FEA0  ACE_OS_Thread_Adapter::invoke+D9  d:\compilling_wow_2.4.3\dep\acelite\ace\os_thread_adapter.cpp line 86
0000000180054BD8  000000000415FEE0  ace_thread_adapter+38  d:\compilling_wow_2.4.3\dep\acelite\ace\base_thread_adapter.cpp line 120
00000000102072E5  000000000415FF30  _beginthreadex+2D5
00000000102072A4  000000000415FF70  _beginthreadex+294
0000000078D6B6DA  000000000415FFA0  BaseThreadStart+3A

Call stack:
Address   Frame     Function      SourceFile
0000000078EF0A2A  0000000004AEFA50  ZwWaitForSingleObject+A
0000000078D704BF  0000000004AEFAF0  WaitForSingleObjectEx+DF
0000000180109D2D  0000000004AEFB30  ACE_OS::cond_wait+AD  d:\compilling_wow_2.4.3\dep\acelite\ace\os_ns_thread.cpp line 1764
0000000180109A00  0000000004AEFBD0  ACE_OS::cond_timedwait+60  d:\compilling_wow_2.4.3\dep\acelite\ace\os_ns_thread.cpp line 1620
000000018006D98C  0000000004AEFC00  ACE_Condition_Thread_Mutex::wait+3C  d:\compilling_wow_2.4.3\dep\acelite\ace\condition_thread_mutex.cpp line 101
000000018006D9DB  0000000004AEFC30  ACE_Condition_Thread_Mutex::wait+3B  d:\compilling_wow_2.4.3\dep\acelite\ace\condition_thread_mutex.cpp line 108
000000018002867D  0000000004AEFC70  ACE_Message_Queue<ACE_MT_SYNCH>::wait_not_empty_cond+5D  d:\compilling_wow_2.4.3\dep\acelite\ace\message_queue_t.cpp line 1744
0000000180026100  0000000004AEFCE0  ACE_Message_Queue<ACE_MT_SYNCH>::dequeue_head+C0  d:\compilling_wow_2.4.3\dep\acelite\ace\message_queue_t.cpp line 1936
000000018003587E  0000000004AEFD30  ACE_Activation_Queue::dequeue+4E  d:\compilling_wow_2.4.3\dep\acelite\ace\activation_queue.cpp line 89
00000001407A9A55  0000000004AEFDD0  SqlDelayThread::run+85  d:\compilling_wow_2.4.3\src\shared\database\sqldelaythread.cpp line 39
00000001407A2908  0000000004AEFE10  ACE_Based::Thread::ThreadTask+38  d:\compilling_wow_2.4.3\src\shared\threading.cpp line 187
0000000180114E89  0000000004AEFEA0  ACE_OS_Thread_Adapter::invoke+D9  d:\compilling_wow_2.4.3\dep\acelite\ace\os_thread_adapter.cpp line 86
0000000180054BD8  0000000004AEFEE0  ace_thread_adapter+38  d:\compilling_wow_2.4.3\dep\acelite\ace\base_thread_adapter.cpp line 120
00000000102072E5  0000000004AEFF30  _beginthreadex+2D5
00000000102072A4  0000000004AEFF70  _beginthreadex+294
0000000078D6B6DA  0000000004AEFFA0  BaseThreadStart+3A

Call stack:
Address   Frame     Function      SourceFile
0000000078EF0A2A  000000002E0DFA00  ZwWaitForSingleObject+A
0000000078D704BF  000000002E0DFAA0  WaitForSingleObjectEx+DF
0000000180109D2D  000000002E0DFAE0  ACE_OS::cond_wait+AD  d:\compilling_wow_2.4.3\dep\acelite\ace\os_ns_thread.cpp line 1764
0000000180109A00  000000002E0DFB80  ACE_OS::cond_timedwait+60  d:\compilling_wow_2.4.3\dep\acelite\ace\os_ns_thread.cpp line 1620
000000018006D98C  000000002E0DFBB0  ACE_Condition_Thread_Mutex::wait+3C  d:\compilling_wow_2.4.3\dep\acelite\ace\condition_thread_mutex.cpp line 101
000000018006D9DB  000000002E0DFBE0  ACE_Condition_Thread_Mutex::wait+3B  d:\compilling_wow_2.4.3\dep\acelite\ace\condition_thread_mutex.cpp line 108
000000018002867D  000000002E0DFC20  ACE_Message_Queue<ACE_MT_SYNCH>::wait_not_empty_cond+5D  d:\compilling_wow_2.4.3\dep\acelite\ace\message_queue_t.cpp line 1744
0000000180026100  000000002E0DFC90  ACE_Message_Queue<ACE_MT_SYNCH>::dequeue_head+C0  d:\compilling_wow_2.4.3\dep\acelite\ace\message_queue_t.cpp line 1936
000000018003587E  000000002E0DFCE0  ACE_Activation_Queue::dequeue+4E  d:\compilling_wow_2.4.3\dep\acelite\ace\activation_queue.cpp line 89
00000001407AD6F1  000000002E0DFD30  DelayExecutor::svc+61  d:\compilling_wow_2.4.3\src\shared\delayexecutor.cpp line 69
000000018018296C  000000002E0DFD80  ACE_Task_Base::svc_run+5C  d:\compilling_wow_2.4.3\dep\acelite\ace\task.cpp line 271
0000000180183F7C  000000002E0DFE10  ACE_Thread_Adapter::invoke_i+CC  d:\compilling_wow_2.4.3\dep\acelite\ace\thread_adapter.cpp line 145
0000000180183E73  000000002E0DFEA0  ACE_Thread_Adapter::invoke+F3  d:\compilling_wow_2.4.3\dep\acelite\ace\thread_adapter.cpp line 94
0000000180054BD8  000000002E0DFEE0  ace_thread_adapter+38  d:\compilling_wow_2.4.3\dep\acelite\ace\base_thread_adapter.cpp line 120
00000000102072E5  000000002E0DFF30  _beginthreadex+2D5
00000000102072A4  000000002E0DFF70  _beginthreadex+294
0000000078D6B6DA  000000002E0DFFA0  BaseThreadStart+3A

Call stack:
Address   Frame     Function      SourceFile
0000000078EF13EA  000000002EA62130  NtGetContextThread+A
CCCCCCCCCCCCCCCC  000000002EA62138  0000:0000000000000000 
CCCCCCCCCCCCCCCC  000000002EA62140  0000:0000000000000000 
CCCCCCCCCCCCCCCC  000000002EA62148  0000:0000000000000000 
0000000000000378  000000002EA62150  
```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Zaffy
**Original Date:** 04.02.2014 00:37:32 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8068748

<hr>

Try to build in Release mode. If I my analysis is correct this issue should disappear.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** digger
**Original Date:** 04.02.2014 01:04:23 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8068908

<hr>

Thank you for not leaving the problem unattended, updated to "939adb2 M 
Merged in Zaffy / oregoncore (pull request # 62) "compiled and tried to cause a crash, the crash was not until everything works. Compile just two patches" pvprank.patch "and" renamenpc.patch

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** LordUsagi
**Original Date:** 04.02.2014 01:20:35 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/491/#comment-8069033

<hr>

Due to custom patches.
